types/recharts/index.d.ts

```diff
@@ -460,7 +461,7 @@ export interface RadialBarProps {
 	data: any[];
 	isAnimationActive?: boolean;
 	animationBegin?: number;
-	animationDuration?: number;
+	// animationDuration?: number;
```

Comment:
If the property is not there any more, could you please *remove* it instead of commenting it out?